### ISSUE TITLE
Develop --> To merge into master (20/02)

### ISSUE THREAD

**Issue by fincamd** [Arquisoft/viade_en3a#14]
As requested in lab classes, by February the 20th we should have merged develop into master so that it can be checked.

**Comment by Drastur** [comment]
Hi guys,
I was thinking that we should have every part of the documentation wrapped up today, so tomorrow we can take a look at it and check that eveything is ok to be merged into master.

**Comment by PabloFerMar** [comment]
Parts 4, 5 and 6 will be done today I think, waiting on @fincamd so I can wrap a couple of things on my side and merge those 3 parts.

**Comment by fincamd** [comment]
Hello there,

I am going to create a pull request from development to master so we have it ready to be merged for tomorrow's lab session. Any change made on development will be included as well so that we can check failure right before merging for revision.
